x-pack/test/detection_engine_api_integration/basic/tests/patch_rules_bulk.ts

```diff
@@ -0,0 +1,356 @@
+import {
+  createSignalsIndex,
+  deleteAllAlerts,
+  deleteSignalsIndex,
+  getSimpleRule,
+  getSimpleRuleOutput,
+  removeServerGeneratedProperties,
+  getSimpleRuleOutputWithoutRuleId,
+  removeServerGeneratedPropertiesIncludingRuleId,
+} from '../../utils';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext) => {
+  const supertest = getService('supertest');
+  const es = getService('legacyEs');
+
+  describe('patch_rules_bulk', () => {
+    describe('patch rules bulk', () => {
+      beforeEach(async () => {
+        await createSignalsIndex(supertest);
+      });
+
+      afterEach(async () => {
+        await deleteSignalsIndex(supertest);
+        await deleteAllAlerts(es);
+      });
+
+      it('should patch a single rule property of name using a rule_id', async () => {
+        // create a simple rule
+        await supertest
+          .post(DETECTION_ENGINE_RULES_URL)
+          .set('kbn-xsrf', 'true')
+          .send(getSimpleRule('rule-1'))
+          .expect(200);
+
+        // patch a simple rule's name
+        const { body } = await supertest
+          .patch(`${DETECTION_ENGINE_RULES_URL}/_bulk_update`)
+          .set('kbn-xsrf', 'true')
+          .send([{ rule_id: 'rule-1', name: 'some other name' }])
+          .expect(200);
+
+        const outputRule = getSimpleRuleOutput();
+        outputRule.name = 'some other name';
+        outputRule.version = 2;
+        const bodyToCompare = removeServerGeneratedProperties(body[0]);
+        expect(bodyToCompare).to.eql(outputRule);
+      });
+
+      it('should patch two rule properties of name using the two rules rule_id', async () => {
+        // create a simple rule
+        await supertest
+          .post(DETECTION_ENGINE_RULES_URL)
+          .set('kbn-xsrf', 'true')
+          .send(getSimpleRule('rule-1'))
+          .expect(200);
+
+        // create a second simple rule
+        await supertest
+          .post(DETECTION_ENGINE_RULES_URL)
+          .set('kbn-xsrf', 'true')
+          .send(getSimpleRule('rule-2'))
+          .expect(200);
+
+        // patch both rule names
+        const { body } = await supertest
+          .patch(`${DETECTION_ENGINE_RULES_URL}/_bulk_update`)
+          .set('kbn-xsrf', 'true')
+          .send([
+            { rule_id: 'rule-1', name: 'some other name' },
+            { rule_id: 'rule-2', name: 'some other name' },
+          ])
+          .expect(200);
+
+        const outputRule1 = getSimpleRuleOutput();
+        outputRule1.name = 'some other name';
+        outputRule1.version = 2;
+
+        const outputRule2 = getSimpleRuleOutput('rule-2');
+        outputRule2.name = 'some other name';
+        outputRule2.version = 2;
+
+        const bodyToCompare1 = removeServerGeneratedProperties(body[0]);
+        const bodyToCompare2 = removeServerGeneratedProperties(body[1]);
+        expect([bodyToCompare1, bodyToCompare2]).to.eql([outputRule1, outputRule2]);
```

Comment:
Nit: these array assertions read a little odd in these tests and I would imagine that the test failure is even more cryptic. My preference would be to make this two separate assertions.

Reply:
Done, changed this one and the others like this to be more clear. Appreciate the honest feedback and time you take to look over details. Always makes the code better!